src/alasqlparser.jison

```diff
@@ -104,13 +105,21 @@ COLUMNS 										return 'COLUMN'
 "CROSS"											return 'CROSS'
 'CUBE'											return 'CUBE'
 "CURRENT_TIMESTAMP"								return 'CURRENT_TIMESTAMP'
+"GETDATE"										return 'CURRENT_TIMESTAMP'
+"NOW"											return 'CURRENT_TIMESTAMP'
 "CURRENT_DATE"									return 'CURRENT_DATE'
 "CURDATE"										return 'CURRENT_DATE'
 "CURSOR"										return 'CURSOR'
 DATABASE(S)?									return 'DATABASE'
+'DAY'											return 'DAY'
+'DAYOFMONTH'									return 'DAY'
+'DAYOFYEAR'										return 'DAY'

```

Comment:
This surely can't be the same as `DAYOFMONTH`?

Reply:
You're right, i corrected this as `DAYOFYEAR` is a separate function.